### PR TITLE
Add properties Help, ValidateStatus and HasFeedback to FormItem

### PR DESCRIPTION
**Pull request by JamesGit-hash**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/form/FormItem.razor.cs [hunk 1]
```diff
@@ -107,6 +108,31 @@ public partial class FormItem : AntDomComponentBase, IFormItem
         [Parameter]
         public FormValidationRule[] Rules { get; set; }
 
+        [Parameter]
+        public bool HasFeedback { get; set; }
+
+        [Parameter]
+        public string ValidateStatus { get; set; }
```
Comment (ElderJames) [inline components/form/FormItem.razor.cs:115]: Please use a enum

File: components/form/FormItem.razor.cs [hunk 2]
```diff
@@ -175,8 +175,8 @@ protected void SetClass()
                 .If($"{_prefixCls}-with-help {_prefixCls}-has-error", () => _isValid == false)
                 .If($"{_prefixCls}-rtl", () => RTL)
                 .If($"{_prefixCls}-has-feedback", () => HasFeedback)
-                .If($"{_prefixCls}-is-validating", () => ValidateStatus == "validating")
-                .GetIf(() => $"{_prefixCls}-has-{ValidateStatus}", () => !string.IsNullOrEmpty(ValidateStatus) && ValidateStatus != "validating" && _iconMap.ContainsKey(ValidateStatus))
+                .If($"{_prefixCls}-is-validating", () => ValidateStatus == FormValidateStatus.Validating)
+                .GetIf(() => $"{_prefixCls}-has-{ValidateStatus.ToString().ToLower()}", () => ValidateStatus == FormValidateStatus.Success || ValidateStatus == FormValidateStatus.Error || ValidateStatus == FormValidateStatus.Warning)
```
Comment (ElderJames) [inline components/form/FormItem.razor.cs:179]: You can use `ValidateStatus.IsIn(...)`

File: components/form/FormItem.razor.cs [hunk 3]
```diff
@@ -107,6 +108,31 @@ public partial class FormItem : AntDomComponentBase, IFormItem
...
+        [Parameter]
+        public bool HasFeedback { get; set; }
+
+        [Parameter]
+        public FormValidateStatus ValidateStatus { get; set; }
+
+        [Parameter]
+        public string Help { get; set; }
+
+        private class IconProperties
```
Comment (ElderJames) [inline components/form/FormItem.razor.cs:120]: You can use ValueTuple

File: components/form/FormItem.razor.cs (hunk 1 above)
Comment (JamesGit-hash) [reply components/form/FormItem.razor.cs]: done.

File: site/AntDesign.Docs/Demos/Components/Form/demo/validate-static.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      <Input.Password placeholder="with input password" />
+    </Form.Item>
+
+    <Form.Item label="Error" hasFeedback validateStatus="error">
+      <Input.Password allowClear placeholder="with input password and allowClear" />
+    </Form.Item>
+  </Form>,
+  mountNode,
+);
+```
```
[the remark this answers is not in the document]
Comment (JamesGit-hash) [reply site/AntDesign.Docs/Demos/Components/Form/demo/validate-static.md]: removed.

File: components/form/FormItem.razor.cs (hunk 3 above)
Comment (JamesGit-hash) [reply components/form/FormItem.razor.cs]: have changed to ValueTuple. thanks for the advice.

File: components/form/FormItem.razor.cs (hunk 2 above)
Comment (JamesGit-hash) [reply components/form/FormItem.razor.cs]: done. thanks.

**Review by ElderJames — APPROVED** [review]

**Pull request merged**